src/components/shareable-url-modal/shareable-url-modal.js

```diff
@@ -111,6 +149,18 @@ const ShareableUrlModal = ({ onToggleModal, visible }) => {
       if (request.ok) {
         setResponseUrl(response.url);
         setDeploymentState('success');
+        displayPublishedView();
+
+        // const hostingPlatformVal = {};
+        // if (hostingPlatforms.hasOwnProperty(inputValues.platform)) {
+        //   hostingPlatformVal[inputValues.platform] = { ...inputValues };
+        // }
+        // saveLocalStorage(localStorageSharableUrl, hostingPlatformVal);
+        // const newState = {
+        //   ...hostingPlatformLocalStorageVal,
+        //   ...hostingPlatformVal,
+        // };
+        // setHostingPlatformLocalStorageVal(newState);
```

Comment:
Please remove the comments if not needed.

Reply:
once Step approves the design I'll move the logic back here

Reply:
done